uber/model_checks.py

```diff
@@ -74,7 +74,7 @@ def group_money(group):
         amount_refunded = int(float(group.amount_refunded))
```

Comment:
It's a little janky, but it's because of this:

```python
>>> int('5.0')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '5.0'
```

To get around that, we've traditionally done ``int(float(s))``.  The downside is that if someone enters ``5.25`` then that will get parsed as just ``5``.  Since we never give fractional refunds that shouldn't be a problem here, so we've never worried about it.  (Even if we did ever give a fractional refund, those pennies would be, to use a term I learned from one of our accounts, "not materially significant".)